src/bn_txns.erl

```diff
@@ -55,7 +55,21 @@ follower_height(#state{db = DB, default = DefaultCF}) ->
     end.
 
 load_chain(_Chain, State = #state{}) ->
-    {ok, State}.
+    maybe_load_genesis(State).
+
+maybe_load_genesis(State = #state{}) ->
+    case blockchain:get_block(1, blockchain_worker:blockchain()) of
```

Comment:
why not use the Chain that is passed in `load_chain` higher up?

Reply:
Ah yes. Good catch.